### PR TITLE
reworked jwt logic with authentication

### PR DESCRIPTION
**Pull request by BatiGencho**
-  rename authrpc jwt dir to something btc server related
-  fix: can bypass btc server authentication with grpcurl

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -448,7 +447,7 @@ start-btc-server-1:
 	--bitcoind-url "${BITCOIND_URL}" \
 	--bitcoind-user "${BITCOIND_USER}" \
 	--bitcoind-pass "${BITCOIND_PWD}" \
-	--jwt-secret "${NODE_1_DIR}/jwt.hex" \
+	--jwt-secret "${NODE_1_DIR}/bjwt.hex" \
```
Comment (arminsabouri) [inline Makefile]: I would have a more verbose name here. Like bitcoin-signing-server-jwt

Comment (arminsabouri) [reply Makefile]: Cc @birozuru for changes to infra docs

**Comment by scottmillner** [comment]
looks good!
will wait for armins comment to be addressed then will approve

File: Makefile (hunk 1 above)
Comment (BatiGencho) [reply Makefile]: fixed all here

**Comment by arminsabouri** [comment]
I also want to manually test this.

**Review by arminsabouri — DISMISSED** [review]
TAck [33c138d](https://github.com/botanix-labs/Macbeth/pull/283/commits/33c138d7461487b536a86abb014054e52e6de2f8)

**Review by arminsabouri — APPROVED** [review]

**Pull request merged**